dist/css/general.css

```diff
@@ -35,6 +35,12 @@
  * Solution to hidden elements readable by screen reader:
  * http://a11yproject.com/posts/how-to-hide-content/
  */
+/**
+* 1 - Define class name, eg `steps--count3`;
+* 2 - Except first child, all steps must be margin-left relative to wrapper and
+      quantity of node steps;
+* 3 - Define line after each step, except to the last child.
+*/
```

Comment:
this comments are in the compiled CSS files, @shankarcabus ... the SASS compiler is keeping these comments in this weird order :S